bukkit/src/main/java/com/github/games647/commandforward/bukkit/CommandForwardBukkit.java

```diff
@@ -26,22 +26,45 @@ public void onEnable() {
     @Override
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
         if (args.length == 0 || args.length == 1) {
-            sender.sendMessage(ChatColor.DARK_RED + "Command is missing");
+            sender.sendMessage(ChatColor.RED + "[CommandForward] Command is missing");
```

Comment:
Please use plugin.getName, so the name of the plugin could be changed from a single source.